src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/CommentsIndentationCheck.java

```diff
@@ -346,7 +346,7 @@ private static boolean isInEmptyCaseBlock(DetailAST prevStmt, DetailAST nextStmt
     }
 
     /**
-     * Checks whether comment is a 'fall through' comment.
+     * Checks whether comment is a 'Fall\ through' comment.
```

Comment:
revert all changes from this file.
your targets are only xxxxxx/fallthrough/InputFallThroughXxxxx.java

Reply:
ok

Reply:
Revert all changes in this file.